packages/aws-cdk/lib/init.ts

```diff
@@ -5,6 +5,7 @@ import * as colors from 'colors/safe';
 import * as fs from 'fs-extra';
 import { error, print, warning } from './logging';
 import { cdkHomeDir } from './util/directories';
+import { versionNumber } from './version';
```

Comment:
can't find version.ts in this PR?

Reply:
It's not part of this PR; it's existing code. https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/lib/version.ts